### DISCUSSION TITLE
12-05

### DISCUSSION THREAD

**Discussion by hanseulhee** [ChristmasTest/christmas#41] [Daily_Meeting]
### 🎅🏻 회의 내용

테스트 코드 발표
-
- 수영님 : 따로 프로젝트 만들어서 해봤다 그저 고수.... ( 결과 : 성공! ) 수영님 유산소 안함 ( 노산소 )
- 윤혁님 : ^^...
- 수님 : pt 100만원!!!! 역시 서울, 슬희님 많이 한다니까 불탐, 오늘 2시간 할 예정, 쓰러진다, 1시간 30분만 해야지
- 슬희님 : 주 5일 운동함. 1시간 30분 ~ 2시간 헬짱, 크로마틱 / 스토리북 ci 개발함
- 귀여운 커서에, 움직일 때마다 눈가루 샤라락 뿌리는 거 어때요? ❄️ > ( 모바일 전용이라 흠... ) (시간남으면 하자)

### ❄️ 오늘 할 일

낼까지 몽고디비 공부해오기 (별로다 ? msw)
수영님 테스트 코드 세팅 pr

### ⛄️ TMI

슬 : 팀미팅 쏘리 ... 아침 병원 예약
수 : 의욕 떨어짐 ㅠ ㅠ, 오늘 하체 할 것임
수영 : 1) 피부가 조아지는 중 ㅋ, 2) 크리스마스 케이크 사줌
윤혁 : 무게 늘어따 ㅋ

**Comment by oauch** [comment]
네 맞아요 ㅋㅋㅋㅋㅋㅋ
